.github/release.yaml

```diff
@@ -0,0 +1,7 @@
+changelog:
+  exclude:
+    authors:
+      - dependabot
+      - pre-commit-ci
+      - github-actions
+      - meeseeksmachine
```

Comment:
I'd rather have this one in there so that the backports do show up. I'm not too worried about it being the sole author if it does open all the PRs.
```suggestion
```